src/hotspot/share/opto/stringopts.cpp

```diff
@@ -39,6 +39,19 @@
 
 #define __ kit.
 
+static bool is_SB_toString(Node* call) {
+  if (call->is_CallStaticJava()) {
+    CallStaticJavaNode* csj = call->as_CallStaticJava();
+    ciMethod* m = csj->method();
+    if (m != NULL &&
+        (m->intrinsic_id() == vmIntrinsics::_StringBuilder_toString ||
+          m->intrinsic_id() == vmIntrinsics::_StringBuffer_toString)) {
+      return true;
+    }
+  }
+  return false;
+}
+
```

Comment:
Was there a reason to shift the order of this function with respect to class StringConcat declaration?  The change in ordering makes it more difficult to spot the differences, if any.